### ISSUE TITLE
Leopard Issue: 

### ISSUE THREAD

**Issue by dandolodavid** [Picovoice/leopard#266]
Make sure you have read the documentation, and have put forth a reasonable effort to find an existing answer.

### Expected behaviour
I've download from the folder https://github.com/Picovoice/leopard/tree/master/lib/common the default models.
I put all of them into the config/language_models folder. Then I try to load them with the model path, but an error occurs

pvleopard.create(access_key='pvK4UQaH54rtAcYZed1CnZ4h/G7vobgu9KcsF+ZUa1OpxbS4rq12iQ==',
                model_path = 'config/language_models/leopard_params.pv')

### Actual behaviour
---------------------------------------------------------------------------
LeopardInvalidArgumentError               Traceback (most recent call last)
/var/folders/n0/7rhb7mg55rdfrz13sxnn0dgc0000gn/T/ipykernel_14326/2304799399.py in <module>
----> 1 pvleopard.create(access_key=myKey,
      2                 model_path = 'config/language_models/leopard_params.pv')

~/opt/anaconda3/lib/python3.9/site-packages/pvleopard/_factory.py in create(access_key, model_path, library_path, enable_automatic_punctuation)
     38         library_path = default_library_path('')
     39 
---> 40     return Leopard(
     41         access_key=access_key,
     42         model_path=model_path,

~/opt/anaconda3/lib/python3.9/site-packages/pvleopard/_leopard.py in __init__(self, access_key, model_path, library_path, enable_automatic_punctuation)
    156         status = init_func(access_key.encode(), model_path.encode(), enable_automatic_punctuation, byref(self._handle))
    157         if status is not self.PicovoiceStatuses.SUCCESS:
--> 158             raise self._PICOVOICE_STATUS_TO_EXCEPTION[status]()
    159 
    160         self._delete_func = library.pv_leopard_delete

LeopardInvalidArgumentError: 

### Steps to reproduce the behaviour
(Include enough details so that the issue can be reproduced independently.)

### Additional info
I've insalled the package using pip, actual version pvleopard==1.2.2
Also, I tried with a download model from the picovice console, but the result is the same error.
The code is correct, because if I copy in the same folder the default model downloaded with the package itself, it's working.

**Comment by kenarsa** [comment]
1- please don't assign a `bug` label until you are sure it's a bug.
2- don't put your AccessKey in clear text publicly.
3- I find it really hard to follow the error messages you have here and I have no way to reproduce the error. closing because of this. if you want to provide more information and provide a stack trace that is readable feel free to re-open.
